src/components/Missions/MyProfile.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-console */
```

Comment:
- [ ] The app returns a console.log to the console after getting the data or changing the view, kindly remove any console.log statement from your code, it will slow your app and reduce performance